lib/game.rb

```diff
@@ -12,14 +12,14 @@ class Game
               :venue_link
 
   def initialize(details)
-    @game_id = details[:game_id].to_i
-    @season = details[:season].to_i
+    @game_id = details[:game_id]
```

Comment:
Based on the spec harness, we realized that we did not need to change any elements of any singular class to an integer value in the initialization.